origami/types/access_levels.py

```diff
@@ -103,6 +103,9 @@ def _generate_next_value_(name, start, count, last_values):
     # the private visibility is the default visibility and does not
     # grant any implicit access level for a resource
     private = enum.auto()
+    # public visibility allows any user to access the resource.
+    # this includes anonymous users or users without a Noteable account.
+    public = enum.auto()
```

Comment:
I just noticed this difference when a schema error was thrown. This change is irrelevant to this PR.